tools/ogs/identifySubdomains.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="identifysubdomains" name="OGS IdentifySubdomains" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>
+        Checks if subdomain meshes are part of the bulk mesh and annotates them with bulk node and element IDs.
```

Comment:
Should be as short as possible. Maybe `Annotate subdomain meshes with bulk node and element IDs`? If needed double check with Lars.